lib/salus/repo.rb

```diff
@@ -51,14 +53,20 @@ class Repo
       { handle: :hardhat_config_ts, filename: 'hardhat.config.ts' }
     ].freeze
 
+    # Define config for RepoSearcher
+    config = { "pass_on_raise" => false, "scanner_timeout_s" => 0 }
```

Comment:
Why was this added?

Reply:
a config is required to initialize a `RepoSearcher`, this was the default